files/build_templates/docker_image_ctl.j2

```diff
@@ -118,25 +118,48 @@ function preStartAction()
 
 function setPlatformLagIdBoundaries()
 {
-    CHASSIS_CONF=/usr/share/sonic/device/$PLATFORM/chassisdb.conf
-    if [ -f "$CHASSIS_CONF" ]; then
-        source $CHASSIS_CONF
-        docker exec -i ${DOCKERNAME} $SONIC_DB_CLI CHASSIS_APP_DB SET "SYSTEM_LAG_ID_START" "$lag_id_start"
-        docker exec -i ${DOCKERNAME} $SONIC_DB_CLI CHASSIS_APP_DB SET "SYSTEM_LAG_ID_END" "$lag_id_end"
-    fi
+    docker exec -i ${DOCKERNAME} $SONIC_DB_CLI CHASSIS_APP_DB SET "SYSTEM_LAG_ID_START" "$lag_id_start"
+    docker exec -i ${DOCKERNAME} $SONIC_DB_CLI CHASSIS_APP_DB SET "SYSTEM_LAG_ID_END" "$lag_id_end"
 }
 {%- endif %}
 
 function postStartAction()
 {
 {%- if docker_container_name == "database" %}
+    CHASSISDB_CONF="/usr/share/sonic/device/$PLATFORM/chassisdb.conf"
+    [ -f $CHASSISDB_CONF ] && source $CHASSISDB_CONF
     if [ "$DEV" ]; then
         # Enable the forwarding on eth0 interface in namespace.
         SYSCTL_NET_CONFIG="/etc/sysctl.d/sysctl-net.conf"
         docker exec -i database$DEV sed -i -e "s/^net.ipv4.conf.eth0.forwarding=0/net.ipv4.conf.eth0.forwarding=1/;
                                                s/^net.ipv6.conf.eth0.forwarding=0/net.ipv6.conf.eth0.forwarding=1/" $SYSCTL_NET_CONFIG
         docker exec -i database$DEV sysctl --system -e
         link_namespace $DEV
+
+
+        if [[ -n "$midplane_subnet" ]]; then
+           # Use /16 for loopback interface
```

Comment:
I dont see code to cleanup all these linux networking state when database docker restart?.